specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -7482,6 +7482,27 @@
       },
       "description": "The configuration parameters used while performing a rolling upgrade."
     },
+    "ScaleInPolicy": {
```

Comment:
the subsequent property name `scaleInPolicy` is not consistent with this one which is capitalized.

Reply:
That's how it is for all properties.